README.md

```diff
@@ -88,6 +92,38 @@ $ export updates_repo_url="<updates_repo_url_to_package_updates>"
 $ make rhel_iso
 ```
 
+<a name="releasing-minishift-iso"></a>
+## Releasing Minishift ISO
+
+<a name="create-a-release-notes-pr"></a>
+### Create a Release Notes PR
+
+Assemble all the meaningful changes since the last release into the CHANGELOG.md file.
+See [this PR](https://github.com/kubernetes/minikube/pull/164) for an example.
+
+<a name="bump-the-version-in-the-makefile"></a>
+### Bump the version in the Makefile
+
+See [this PR](https://github.com/kubernetes/minikube/pull/165) for an example.
```

Comment:
This is not inline with what you did. You did define the version in a separate VERSION file. I'd prefer to have the version in the Makefile as well. BTW, I don't think we need to reference a minikube pull request here.